src/NexusMods.Library/InstallLoadoutItemJobWorker.cs

```diff
@@ -34,8 +34,11 @@ protected override async Task<JobResult> ExecuteAsync(InstallLoadoutItemJob job,
 
         var result = await ExecuteInstallersAsync(job, installers, cancellationToken);
 
-        if (!result.HasValue && job.Installer is not AdvancedManualInstaller)
+        if (!result.HasValue)
         {
+            if (job.Installer is not AdvancedManualInstaller)
+                return JobResult.CreateFailed($"Advanced installer did not succeed for `{job.LibraryItem.Name}` (`{job.LibraryItem.Id}`)");
```

Comment:
This doesn't sound right though, if the last installer is _not_ the Advanced one and the result is empty we should attempt using the Advanced installer.

Reply:
erri beat me to it